pkg/apis/core/v1/defaults.go

```diff
@@ -429,3 +441,13 @@ func SetDefaults_HostPathVolumeSource(obj *v1.HostPathVolumeSource) {
 		obj.Type = &typeVol
 	}
 }
+
+// isRestartableInitContainer returns true if the initContainer has
+// ContainerRestartPolicyAlways.
+func isRestartableInitContainer(initContainer *v1.Container) bool {
```

Comment:
Can we reuse the podutil function here too?

Reply:
Same comment as above: https://github.com/kubernetes/kubernetes/pull/128367#discussion_r1820043897.